easybuild/easyconfigs/t/THetA/THetA-0.7-GCC-8.2.0-2.31.1-Python-2.7.15.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CmdCp'
+
+name = 'THetA'
+version = '0.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/raphael-group/THetA'
+description = """Tumor Heterogeneity Analysis (THetA) and THetA2 are algorithms
+ that estimate the tumor purity and clonal/subclonal copy number aberrations
+ directly from high-throughput DNA sequencing data."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+source_urls = ['https://github.com/raphael-group/THetA/archive/']
+sources = [{'download_filename': 'v%(version)s.tar.gz','filename': SOURCE_TAR_GZ}]
+checksums = ['474452fa87a1ca0228137693bfede3200b91e4e166b2216c4bd24f8b270c1dea']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('MATLAB', '2019a', '', True),
```

Comment:
I'm a bit skeptical about these deps on both Python and MATLAB. The software seems like it would still be usable without them. It seems likely that a user could just just load whichever version of MATLAB they needed. 

Python 2.7.15 also forces python2 on any other modules that might be loaded, which severely limits the usability. I'd say this is just best left up to the user. (perhaps suggest it includes MATLAB and Python bindings in the description?)

Reply:
The main code is in Python and you can see that it is not Python 3 compliant. So there is no choice. 
Matlab is indeed an optional dependency (as it is Java), but I do not see any downside in having a concurrent version of Matlab loaded by default. There's no impact on performance for users not needing it, it is convenient for user using it and if any user really wants a different Matlab, it is always possible to replace the module as there are no toolchain constrains.